src/app/app.component.ts

```diff
@@ -7,4 +8,18 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'splitwise';
+  // Declare showLoginButton property
+  showLoginButton: boolean;
+  constructor(private router: Router) {
+    // Initialize showLoginButton property to true
+    this.showLoginButton=true;
```

Comment:
Make consistency in adding space before and after '='

Reply:
Changes are done